cmake/installdata/windows/dlls.txt

```diff
@@ -1,3 +1,4 @@
 system/*.dll .
+project/BuildDependencies/${ARCH}/bin/libbluray*.jar .
```

Comment:
now you have to add `system/libbluray*.jar .` back to work on win32 (both lines are needed)